docs/config.md

```diff
@@ -4,24 +4,24 @@ Configuration
 You can configure the app by copying `config.sample.json` to
 `config.json` and customising it:
 
-For a good example, see https://riot.im/develop/config.json.
+For a good example, see https;//develop.element.io/config.json.
```

Comment:
There is a typo here: `https;//` -> `https://`

Reply:
Dang, good catch